ui/i18n/src/main/res/values/strings.xml

```diff
@@ -93,6 +93,8 @@
 
     <!-- Playback history -->
     <string name="clear_history_label">Clear History</string>
+    <string name="clear_playback_history">Clear playback history</string>
```

Comment:
Why not just re-use the string above? That ensures consistency in the translated strings and avoids additional work for the translators